specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2023-05-01/amlfilesystem.json

```diff
@@ -0,0 +1,1145 @@
+                "read",
+                "create"
+              ],
+              "$ref": "storagecache.json#/definitions/URLString"
+            },
+            "clientInfo": {
+              "$ref": "#/definitions/AmlFilesystemClientInfo",
+              "description": "Client information for the AML file system."
+            },
+            "throughputProvisionedMBps": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Throughput provisioned in MB per sec, calculated as storageCapacityTiB * per-unit storage throughput",
+              "readOnly": true
+            },
+            "encryptionSettings": {
+              "$ref": "#/definitions/AmlFilesystemEncryptionSettings",
+              "description": "Specifies encryption settings of the AML file system."
+            },
+            "maintenanceWindow": {
+              "description": "Start time of a 30-minute weekly maintenance window.",
+              "type": "object",
+              "properties": {
+                "dayOfWeek": {
+                  "type": "string",
+                  "enum": [
+                    "Monday",
+                    "Tuesday",
+                    "Wednesday",
+                    "Thursday",
+                    "Friday",
+                    "Saturday",
+                    "Sunday"
+                  ],
+                  "x-ms-enum": {
+                    "name": "MaintenanceDayOfWeekType",
+                    "modelAsString": false
+                  },
+                  "description": "Day of the week on which the maintenance window will occur."
+                },
+                "timeOfDayUTC": {
+                  "type": "string",
+                  "pattern": "^([0-9]|0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$",
+                  "description": "The time of day (in UTC) to start the maintenance window."
+                }
+              }
+            },
+            "hsm": {
+              "type": "object",
+              "description": "Hydration and archive settings and status",
+              "properties": {
+                "settings": {
+                  "$ref": "#/definitions/AmlFilesystemHsmSettings",
+                  "description": "Specifies HSM settings of the AML file system."
+                },
+                "archiveStatus": {
+                  "type": "array",
+                  "readOnly": true,
+                  "description": "Archive status",
+                  "items": {
+                    "$ref": "#/definitions/AmlFilesystemArchive"
+                  },
+                  "x-ms-identifiers": [
+                    "filesystemPath"
+                  ]
+                }
+              }
+            }
+          },
+          "type": "object",
+          "required": [
+            "filesystemSubnet",
+            "storageCapacityTiB",
+            "maintenanceWindow"
+          ]
+        }
+      }
+    },
+    "AmlFilesystemUpdate": {
```

Comment:
>"AmlFilesystemUp [](http://example.com/codeflow?start=4&length=16)

Track resource update must be able to update sku long with tags

Reply:
@sjanamma ,
A SKU cannot be modified in our service; this is intentional.  We have it's mutability classified accordingly as shown below:

"SkuName": {
     "x-ms-mutability": [
       "read",
       "create"
     ],